R/miaTime.R

```diff
@@ -138,3 +138,50 @@ NULL
 #' @keywords data
 #'
 NULL
+
+#' @title CrestStudyData
+#' @description
+#' The CrestStudyData includes microbiota and metabolite profiling data 
+#' from 78 Indian individuals (40 males, 38 females) and 78 European individuals 
+#' matched for sex, BMI, and age.
+#' Indian subjects were grouped into four diet groups (~20 subjects per group), 
+#' and fecal samples were collected across three seasonal time points.
+#'
+#' The microbiota profiling was performed using HITChip microarray analysis 
+#' (in duplicate), qPCR (in triplicate with eight-point standard curves), and 
+#' LC-HRMS and HPLC metabolite profiling with internal standards.
+#'
+#' Column metadata includes diet group assignment, sampling season, sex, BMI, 
+#' age, and questionnaire-based lifestyle metadata.
```

Comment:
Even though this data comes from publication, we could try to polish it little bit:

1. Harmonize colData columns
Currently, they are with different styles (dry.fish, Hentak..fermentd.fish.paste..boiled..)

--> All lower case and "_" between words

2. Instead of y/n, use TRUE/FALSE (more explicit)

Reply:
3. Age groups:
0-10 10--20 20-30 30-40 40-50 50-60 60-70 70-80 80-90

--> Fix 10--20 --> 10-20